### PR TITLE
Update package-lock.json to v2

### PR DESCRIPTION
**Pull request by Iinh**
[The latest tag](https://github.com/mozilla/glam/releases/tag/2021.11.0) deployment to stage failed. See: https://app.circleci.com/pipelines/github/mozilla/glam/4184/workflows/ca24fe0e-d036-4219-be26-df904b3e8961/jobs/8209

Reading the error log, it looks like the issue is that Docker wasn't able to update to the latest npm package (v8.1.2). This might be due to the fact that our current package-lock.json is outdated.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -4,12 +4,18 @@ GLAM is maintained on GitHub at:
...
-install [Docker] and [Docker Compose] on your computer to contribute code or
-documentation changes. Note that GLAM requires a fair bit of disk space-- if
-you're using Docker for Mac, you might want to increase the allocation of disk
-space to the Docker virtual machine to at least 100G (via Preferences ->
-Resources -> Advanced).
+To install a local copy of GLAM, you need:
+
+- [Python](https://www.python.org/) (version 3.8+)
+- [node.js](https://nodejs.org/) (version 12+)
+- [npm](https://docs.npmjs.com/cli/v7/commands/npm) (v7+): run
```
Comment (wlach) [inline docs/development.md:11]: This says 7+, but package.json says 8. It looks like we only made the Glean Dictionary require 7, and it works fine:

https://github.com/mozilla/glean-dictionary/blob/main/package.json

**Review by wlach — APPROVED** [review]
r+wc just one nit.

File: docs/development.md (hunk 1 above)
Comment (Iinh) [reply docs/development.md:11]: Yup I think using npm v7+ is good enough. Let me update this.

**Comment by Iinh** [comment]
This didn't seem to fix the issue, see [log](https://app.circleci.com/pipelines/github/mozilla/glam/4190/workflows/ad911495-d4a9-4210-8d9b-a5d3c7f75cd0/jobs/8222) 🙈 Further investigation is needed.

**Pull request merged**